crates/store/re_types/definitions/rerun/archetypes/asset_video.fbs

```diff
@@ -2,10 +2,10 @@ namespace rerun.archetypes;
 
 /// A video binary.
 ///
-/// NOTE: Videos can only be viewed in the Rerun web viewer.
-/// Only MP4 containers with a limited number of codecs are currently supported, and not in all browsers.
+/// Only MP4 containers with AV1 is generally supported,
+/// though the web viewer sometimes supports some more video codex, depending on browser.
```

Comment:
```suggestion
/// Only MP4 containers with AV1 are generally supported,
/// though the web viewer support more video codex, depending on browser.
```

I mean they support _a lot_ more and I don't think we'll add any more there ourselves :)